libs/agno/agno/agent/agent.py

```diff
@@ -106,6 +104,8 @@ class Agent:
     session_state: Optional[Dict[str, Any]] = None
     search_previous_sessions_history: Optional[bool] = False
     num_history_sessions: Optional[int] = None
+    # Session summarizer
+    summary_manager: Optional[SessionSummarizer] = None
```

Comment:
```suggestion
    session_summarizer: Optional[SessionSummarizer] = None
```

Reply:
Since the session summarizer is just a model with instructions, we can also make it simpler by just expecting `summarizer_model` and `summarizer_instructions` (optional, we use default if not set)
wdyt?